### PR TITLE
Update README with release info & badges

### PR DESCRIPTION
**Pull request by mocsharp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +21,25 @@ Currently supported message broker services:
...
+
+Official builds are made from the `main` branch.
+
+### RC Builds
+
+Release candidates are built and released from the `release/*` branches.
+
+### Development Builds
+
+Development builds are made from all branches except the `main` branch and the `release/*` branches. The NuGet packages are released to [GitHub](https://github.com/Project-MONAI/monai-deploy-storage/packages/1350678) only.
```
Comment (dbericat) [inline README.md:38]: Messaging. Not storage?

File: README.md [hunk 2]
```diff
@@ -16,9 +21,25 @@ Currently supported message broker services:
 
 If you would like to use a message broker service not listed above, please file an [issue](https://github.com/Project-MONAI/monai-deploy-messaging/issues) and contribute to the repository.
 
+## Releases
+
+The MONAI Deploy Storage library is released in NuGet format, which is available on both [NuGet.Org](https://www.nuget.org/packages/Monai.Deploy.Storage/) and [GitHub](https://github.com/Project-MONAI/monai-deploy-storage/packages/1350678).
```
Comment (dbericat) [inline README.md:26]: Messaging, not storage

**Review by dbericat — CHANGES_REQUESTED** [review]
Couple of copy paste storage vs messaging. Please review

File: README.md (hunk 1 above)
Comment (mocsharp) [reply README.md]: Good catch; fixed

File: README.md (hunk 2 above)
Comment (mocsharp) [reply README.md]: Good catch; fixed

File: README.md [hunk 3]
```diff
@@ -16,9 +21,25 @@ Currently supported message broker services:
...
+Release candidates are built and released from the `release/*` branches.
+
+### Development Builds
+
+Development builds are made from all branches except the `main` branch and the `release/*` branches.  The NuGet packages are released to [GitHub](https://github.com/Project-MONAI/monai-deploy-messaging/packages/1365839) only.
+
 ## Contributing
 
-For guidance on making a contribution to MONAI Deploy Messaging, see the [contributing guidelines](https://github.com/Project-MONAI/monai-deploy/blob/main/CONTRIBUTING.md).
+For guidance on contributing MONAI Deploy Messaging, see the [contributing guidelines](https://github.com/Project-MONAI/monai-deploy/blob/main/CONTRIBUTING.md).
```
Comment (dbericat) [inline README.md:42]: contributing TO?

**Review by dbericat — CHANGES_REQUESTED** [review]
Minor grammar

File: README.md (hunk 3 above)
Comment (mocsharp) [reply README.md]: Fixed

**Review by dbericat — APPROVED** [review]
LGTM

**Pull request merged**